capstone_project/src/main/java/com/google/sps/servlets/MapServlet.java

```diff
@@ -75,20 +74,22 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       return;
     }
 
-    addToDatastore(request);
+    School submission = new Gson().fromJson(request.getReader(), School.class);
```

Comment:
is submission a good name? this is a doGet method. I think requestedSchool or something similar to that would be better